src/credentials/tests/CHIPCert_test_vectors.cpp

```diff
@@ -2528,6 +2528,16 @@ extern constexpr ByteSpan sTestCert_PDCID01_Chip((const uint8_t[]){
     0x2f, 0x91, 0x93, 0xfd, 0x2c, 0xbc, 0x4e, 0x23, 0x50, 0x0a, 0x31, 0x7d, 0xa4, 0xa0, 0x79, 0xb2, 0xdb, 0x69, 0x42, 0x18,
 });
 
+extern constexpr ByteSpan sTestCert_PDCID01_ChipCompact((const uint8_t[]){
```

Comment:
Please add TODO comment with an issue about matching the test case to a spec test vector in spec appendix. SDK code should match validation against spec examples (pattern followed a lot of other places in this test/library)

Reply:
Yeah I’ll add these to the spec

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/7916